src/examples/dropdowns/SelectValidation.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import {
+  Dropdown,
+  Field,
+  Menu,
+  Item,
+  Select,
+  Label,
+  Message
+} from '@zendeskgarden/react-dropdowns';
+
+const StyledSelect = styled(Select)`
+  * {
+    box-sizing: initial;
```

Comment:
The CSS for the select element makes the dropdown chevron icon very small (almost can't see it) when bedrock sets this element's `box-sizing` to `border-box`:

<img width="821" alt="Screen Shot 2020-06-01 at 10 21 38 AM" src="https://user-images.githubusercontent.com/1811365/83435877-ca9db380-a3f1-11ea-8166-0ace8264e28b.png">

When the `box-sizing` is reset to initial, the chevrons become the correct size:

<img width="822" alt="Screen Shot 2020-06-01 at 10 22 04 AM" src="https://user-images.githubusercontent.com/1811365/83435882-cd000d80-a3f1-11ea-93af-0825fcc3badb.png">

Any suggestions for an alternative approach here?

Reply:
This is fixed by https://github.com/zendeskgarden/react-components/pull/756

Reply:
Awesome, I've removed the `StyledSelect`. I expect to update the `react-dropdowns` package once https://github.com/zendeskgarden/react-components/pull/756 is released. That should fix the chevron sizing issue.